Os/test/ut/TestMain.cpp

```diff
@@ -38,9 +38,10 @@ TEST(Nominal, QTestPerformance) {
 TEST(Nominal, QTestConcurrentTest) { 
    qtest_concurrent();
 }
-TEST(Nominal, IntervalTimerTest) { 
-   intervalTimerTest();
-}
+// Disabled for macOS known issue
```

Comment:
I've now actually seen this test fail on linux as well. I think that this test should just be deleted... if we want to unit test interval timers we need to rearchitect the design of interval timers